ConfigurationSystem/Agent/Bdii2CSAgent.py

```diff
@@ -8,15 +8,15 @@
 __RCSID__ = "$Id$"
 
 from DIRAC import S_OK, S_ERROR, gConfig
-from DIRAC.Core.Base.AgentModule import AgentModule
-from DIRAC.Core.Utilities.Grid import getBdiiCEInfo, getBdiiSEInfo
-from DIRAC.FrameworkSystem.Client.NotificationClient import NotificationClient
 from DIRAC.ConfigurationSystem.Client.CSAPI import CSAPI
 from DIRAC.ConfigurationSystem.Client.Helpers.Path import cfgPath
 from DIRAC.ConfigurationSystem.Client.Helpers.Registry import getVOs, getVOOption
+from DIRAC.ConfigurationSystem.Client.Helpers.Resources import getSitesCEsMapping, getCESiteMapping
```

Comment:
getSitesCEsMapping, getCESiteMapping are similar functions for different direction of the same mapping. So, the names should be then similar, I mean: getSitesCEsMapping -> getSiteCEMapping

Reply:
Done.